tools/terraform/rds.tf

```diff
@@ -76,17 +80,20 @@ module "aurora_mysql" {
   create_random_password              = false
   database_name                       = var.database_name
   enable_http_endpoint                = false
-  #performance_insights_enabled        = true
+  performance_insights_enabled        = true
+  enabled_cloudwatch_logs_exports     = ["slowquery"]
 
-  vpc_id                = module.vpc.vpc_id
-  subnets               = module.vpc.database_subnets
-  create_security_group = true
-  allowed_cidr_blocks   = module.vpc.private_subnets_cidr_blocks
+  vpc_id                 = module.vpc.vpc_id
+  vpc_security_group_ids = [aws_security_group.backend.id]
+  subnets                = module.vpc.database_subnets
+  create_security_group  = true
+  allowed_cidr_blocks    = module.vpc.private_subnets_cidr_blocks
 
   replica_count         = 1
   replica_scale_enabled = true
   replica_scale_min     = 1
   replica_scale_max     = 3
+  snapshot_identifier   = "arn:aws:rds:us-east-2:917007347864:cluster-snapshot:fleetdm-mysql-iam-final-prerebuild"
```

Comment:
not sure this will work? copy paste error?

Reply:
No it works. I should add a lifecycle ignore to it and variabilize it.